landing/src/styles/globals.css

```
@@ -373,15 +373,30 @@ html:not([data-anchor-scrolling]) {
-  [data-sidebar-placeholder] {
+  [data-sidebar-placeholder]:not(:has(#nd-sidebar[data-collapsed="true"])) {
     position: fixed !important;
     width: var(--fd-sidebar-width) !important;
     inset-inline-start: max(0px, calc((100vw - var(--fd-layout-width, 97rem)) / 2)) !important;
   }
 }
 
+@media (max-width: 767px) {
+  #nd-subnav {
+    --subnav-height: 3.5rem;
+    position: fixed !important;
```

Stylelint (17.7.0)

[error] Expected empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)